metaflow/current.py

```diff
@@ -15,6 +15,12 @@ def __init__(self):
         self._namespace = None
         self._username = None
         self._is_running = False
+        self._tags = None
+        self._runtime_environment = None
+        self._runtime_name = None
+        self._sfn_state_machine_name = None
```

Comment:
This one seems very plugin specific.

Reply:
I agree, at some point it might make sense for the plugins to have their own implementations of `current` with inheritance of the common attributes? Don't know if that is a huge priority for now

Reply:
good idea, thanks for the tip!

Reply:
This is now resolved :) A related thing I was thinking about is if the environment for `current` is altered by different plugins, what is the best way to surface to users what is accessible from `current`. Right now that takes a bit of digging to work out!